src/bin/nickel.rs

```diff
@@ -62,6 +62,9 @@ enum Command {
         #[structopt(long)]
         history_file: Option<PathBuf>,
     },
+    /// Generate the documentation files for the specified nickel file
+    #[cfg(feature = "doc")]
```

Comment:
We may want to provide a path with a flag like `--dir` or `--out`, but that can wait for another PR.